server/bot/router.py

```diff
@@ -201,3 +209,200 @@ async def delete_bot(
+):
+    bot_id = body.bot_id
+    try:
+        if not user:
+            return JSONResponse(
+                content={
+                    "success": False,
+                    "errorMessage": "您必须先使用 GitHub 登录 Petercat 才能使用此功能。",
+                },
+                status_code=500,
+            )
+        bot_dao = BotDAO()
+        bot = bot_dao.get_bot(bot_id)
+        if not bot:
+            return JSONResponse(
+                content={"success": False, "errorMessage": "机器人不存在,无法公开"},
+                status_code=500,
+            )
+        if not bot["public"]:
+            return JSONResponse(
+                content={
+                    "success": False,
+                    "errorMessage": "机器人未公开，无需重复操作",
+                },
+                status_code=500,
+            )
+        auth = Auth.Token(token=user.access_token)
+        g = Github(auth=auth)
+        repo = g.get_repo(OFFICIAL_REPO)
+        issue_title = f"Market approval: {bot.name}"
+        issue_body = f"""Please remove this robot from the market. Thank you.\n
+| bot_id     | bot_name    | bot_description    |
+|--------------|-----------|-----------|
+| {bot.id} | {bot.name} | {bot.description} |
+        """
+        issue = repo.create_issue(
+            title=issue_title, body=issue_body, labels=["approval"]
+        )
+        bot_approval_dao = BotApprovalDAO()
+        bot_approval = BotApproval(
+            bot_id=bot.id,
+            approval_status=ApprovalStatus.OPEN,
+            task_type=TaskType.MARKET,
+            approval_path=issue.html_url,
+            created_at=datetime.now(),
+        )
+        bot_approval_dao.create(bot_approval)
+        return {"success": True, "approval_path": issue.html_url}
+
+    except Exception as e:
+        return JSONResponse(
+            content={"success": False, "errorMessage": str(e)}, status_code=500
+        )
+
+
+@router.post("/deploy/website", status_code=200)
+async def deploy_bot_to_website(
+    body: BotDeployRequest,
+    user: Annotated[User | None, Depends(get_user)] = None,
+):
+    bot_id = body.bot_id
+    website_url = body.website_url
+    try:
+        if not user:
+            return JSONResponse(
+                content={
+                    "success": False,
+                    "errorMessage": "您必须先使用 GitHub 登录 Petercat 才能使用此功能。",
+                },
+                status_code=500,
+            )
+        bot_dao = BotDAO()
+        bot = bot_dao.get_bot(bot_id)
+        if not bot:
+            return JSONResponse(
+                content={"success": False, "errorMessage": "机器人不存在,无法公开"},
+                status_code=500,
+            )
+        auth = Auth.Token(token=user.access_token)
+        g = Github(auth=auth)
+        repo = g.get_repo(OFFICIAL_REPO)
+        issue_title = f"Website approval : {bot.name}"
+        issue_body = f"""Please have the project administrator review my robot. I would like to deploy the robot on my website. Thank you.\n
```

Comment:
I would like to deploy the robot on my website. Here is the information about my bot.